examples/CustomDataframe/frontend/src/CustomDataframe.tsx

```diff
@@ -18,10 +17,15 @@ interface TableProps {
   element: ArrowTable
 }
 
-class Table extends PureComponent<TableProps> {
+class Table extends React.PureComponent<TableProps> {
+  public componentDidMount = (): void => {
+    // Send the dataframe back to Python.
+    // NOTE: Styler will not be sent back as it's impossible to serialize.
+    Streamlit.setComponentValue(this.props.element.serialize(), "dataframe")
```

Comment:
Is this just for illustrative purposes, or is there a reason you'd want to just return the dataframe back to Streamlit? (If it's just for illustration, I think we should not include it in this example - I think it'll be confusing to users. My first thought would be "is this something I need to do to make dataframes work?")